scylla/src/transport/load_balancing/default.rs

```diff
@@ -1197,6 +1231,19 @@ mod tests {
             let plan = get_plan_and_collect_node_identifiers(policy, routing_info, cluster);
             plans.push(plan);
         }
+        let example_plan = Plan::new(policy, routing_info, cluster);
+        warn!("Example plan from policy:",);
+        for (node, shard) in example_plan {
+            warn!(
+                "Node port: {}, shard: {}, dc: {:?}, rack: {:?}, down: {:?}",
+                node.address.port(),
+                shard,
+                node.datacenter,
+                node.rack,
+                node.is_down()
+            );
+        }
+
```

Comment:
I don't think it's wrong to leave debug logs in tests.
Multiple times (I'm not talking only about the rust driver here) I found myself adding logs to a test I'm debugging (which sometimes takes non-negligible amount of time), removing them later, just to add them back after a day / week / month. This can be frustrating.
Pre-existing logs can aid debugging:
- With good logs we could find the problem from CI output, without needing to debug locally
- They were probably added and not removed later because they were actually useful. Figuring out what you should log in a test can be non-obvious, it's nice to have that (at least partially) done for you.
The cost is just a few lines of code - I'm pretty sure it's a good tradeoff.

Reply:
I can agree with the idea. I still disagree with using the `warn` level, though.